### PR TITLE
🗾 Add Japanese

### PR DESCRIPTION
**Pull request by fu-sen**
relation: https://github.com/GetPublii/Publii-ui-locales/issues/15

I add Japanese.
I don't know the term to translate, so I leave some in English. It will make changes later.

First of all, I hope that the addition of Japanese will increase the number of Publii users in Japan.

### CODE REVIEW

**Comment by rbenitezpagan** [comment]
Thanks @fu-sen!

File: locales/ja/wysiwyg.json [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+    "Cols": "行",
+    "Column": "桁",
+    "Column group": "桁グループ",
+    "Constrain proportions": "Constrain proportions",
+    "Copy": "コピー",
+    "Copy row": "行でコピー",
+    "Could not find the specified string.": "指定された文字列が見つかりませんでした。",
+    "Could not load emoticons": "絵文字を読み込めませんでした",
+    "Count": "Count",
+    "Currency": "Currency",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:63]:
```suggestion
    "Currency": "通貨",
```

File: locales/ja/wysiwyg.json [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+    "Find whole words only": "全文検索のみ",
+    "Finish": "完了",
+    "Flags": "旗・国旗",
+    "Font": "フォント",
+    "Font Sizes": "フォントサイズ",
+    "Fonts": "フォント",
+    "Food and Drink": "食べ物・飲み物",
+    "Footer": "フッター",
+    "Format": "形式",
+    "Format Painter": "Format Painter",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:109]:
```suggestion
    "Format Painter": "フォーマットペインター",
```

File: locales/ja/wysiwyg.json [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+    "Header 6": "見出し H6",
+    "Header cell": "見出しのセル",
+    "Headers": "見出し",
+    "Heading 1": "見出し H1",
+    "Heading 2": "見出し H2",
+    "Heading 3": "見出し H3",
+    "Heading 4": "見出し H4",
+    "Heading 5": "見出し H5",
+    "Heading 6": "見出し H6",
+    "Headings": "Headings",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:132]:
```suggestion
    "Headings": "見出し",
```

File: locales/ja/wysiwyg.json [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+    "No": "いいえ",
+    "No color": "色なし",
+    "Nonbreaking space": "Nonbreaking space",
+    "None": "なし",
+    "Numbered list": "番号付きリスト",
+    "OR": "OR",
+    "Objects": "オブジェクト",
+    "Ok": "Ok",
+    "Open help dialog": "ヘルプダイアログを開く",
+    "Open link in...": "Open link in...",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:209]:
```suggestion
    "Open link in...": "リンクを開く...",
```

File: locales/ja/wysiwyg.json [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+    "Update": "更新",
+    "Upload": "アップロード",
+    "Upper Alpha": "英大文字",
+    "Upper Roman": "ローマ数字大文字",
+    "Url": "URL",
+    "User Defined": "User Defined",
+    "V Align": "V Align",
+    "Valid": "Valid",
+    "Version": "Version",
+    "Vertical space": "Vertical space",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:303]:
```suggestion
    "Vertical space": "垂直方向のスペース",
```

File: locales/ja/wysiwyg.json [hunk 6]
```diff
@@ -0,0 +1,326 @@
...
+    "View": "表示",
+    "Warn": "Warn",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:305]:
```suggestion
    "Warn": "暖かい",
```

File: locales/ja/wysiwyg.json [hunk 7]
```diff
@@ -0,0 +1,326 @@
...
+    "White": "白",
+    "Whole words": "全単語",
+    "Width": "横幅",
+    "Word count": "単語数",
+    "Words": "単語",
+    "Words: {0}": "単語: {0}",
+    "Yellow": "黄色",
+    "Yes": "はい",
+    "You have unsaved changes are you sure you want to navigate away?": "保存されていない変更点があります。削除してよろしいですか？",
+    "alignment": "alignment",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:315]:
```suggestion
    "alignment": "アラインメント",
```

File: locales/ja/wysiwyg.json [hunk 8]
```diff
@@ -0,0 +1,326 @@
...
+    "comments": "コメント",
+    "example": "example",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:317]:
```suggestion
    "example": "例",
```

File: locales/ja/wysiwyg.json [hunk 9]
```diff
@@ -0,0 +1,326 @@
...
+    "formatting": "formatting",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:318]:
```suggestion
    "formatting": "フォーマット",
```

File: locales/ja/wysiwyg.json [hunk 10]
```diff
@@ -0,0 +1,326 @@
...
+    "history": "履歴",
+    "indentation": "インテント",
+    "lowercase": "lowercase",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:321]:
```suggestion
    "lowercase": "小文字",
```

File: locales/ja/wysiwyg.json [hunk 11]
```diff
@@ -0,0 +1,326 @@
...
+    "permanent pen": "permanent pen",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:322]:
```suggestion
    "permanent pen": "油性ペン",
```

File: locales/ja/wysiwyg.json [hunk 12]
```diff
@@ -0,0 +1,326 @@
...
+    "Undo": "元に戻す",
+    "Update": "更新",
+    "Upload": "アップロード",
+    "Upper Alpha": "英大文字",
+    "Upper Roman": "ローマ数字大文字",
+    "Url": "URL",
+    "User Defined": "User Defined",
+    "V Align": "V Align",
+    "Valid": "Valid",
+    "Version": "Version",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:302]:
```suggestion
    "Version": "バージョン",
```

File: locales/ja/wysiwyg.json [hunk 13]
```diff
@@ -0,0 +1,326 @@
...
+    "Underline": "下線",
+    "Undo": "元に戻す",
+    "Update": "更新",
+    "Upload": "アップロード",
+    "Upper Alpha": "英大文字",
+    "Upper Roman": "ローマ数字大文字",
+    "Url": "URL",
+    "User Defined": "User Defined",
+    "V Align": "V Align",
+    "Valid": "Valid",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:301]:
```suggestion
    "Valid": "有効",
```

File: locales/ja/wysiwyg.json [hunk 14]
```diff
@@ -0,0 +1,326 @@
...
+    "Turquoise": "Turquoise",
+    "UPPERCASE": "UPPERCASE",
+    "Underline": "下線",
+    "Undo": "元に戻す",
+    "Update": "更新",
+    "Upload": "アップロード",
+    "Upper Alpha": "英大文字",
+    "Upper Roman": "ローマ数字大文字",
+    "Url": "URL",
+    "User Defined": "User Defined",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:299]:
```suggestion
    "User Defined": "ユーザー定義の",
```

File: locales/ja/wysiwyg.json [hunk 15]
```diff
@@ -0,0 +1,326 @@
...
+    "V Align": "V Align",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:300]:
```suggestion
    "V Align": "V整列",
```

File: locales/ja/wysiwyg.json [hunk 16]
```diff
@@ -0,0 +1,326 @@
...
+    "Text to display": "表示するテキスト",
+    "The URL you entered seems to be an email address. Do you want to add the required mailto: prefix?": "入力された URL は、メールアドレスのようです。mailto: をつけますか？",
+    "The URL you entered seems to be an external link. Do you want to add the required http:// prefix?": "入力されたURLは外部リンクのようです。http:// を付けますか？",
+    "Title Case": "Title Case",
+    "To open the popup, press Shift+Enter": "To open the popup, press Shift+Enter",
+    "Tools": "ツール",
+    "Top": "上",
+    "Travel and Places": "トラベル・場所",
+    "Turquoise": "Turquoise",
+    "UPPERCASE": "UPPERCASE",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:291]:
```suggestion
    "UPPERCASE": "大文字",
```

File: locales/ja/wysiwyg.json [hunk 17]
```diff
@@ -0,0 +1,326 @@
...
+    "Text color": "文字色",
+    "Text to display": "表示するテキスト",
+    "The URL you entered seems to be an email address. Do you want to add the required mailto: prefix?": "入力された URL は、メールアドレスのようです。mailto: をつけますか？",
+    "The URL you entered seems to be an external link. Do you want to add the required http:// prefix?": "入力されたURLは外部リンクのようです。http:// を付けますか？",
+    "Title Case": "Title Case",
+    "To open the popup, press Shift+Enter": "To open the popup, press Shift+Enter",
+    "Tools": "ツール",
+    "Top": "上",
+    "Travel and Places": "トラベル・場所",
+    "Turquoise": "Turquoise",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:290]:
```suggestion
    "Turquoise": "ターコイズ",
```

File: locales/ja/wysiwyg.json [hunk 18]
```diff
@@ -0,0 +1,326 @@
...
+    "Table of Contents": "目次",
+    "Table properties": "テーブルのプロパティ",
+    "Target": "Target",
+    "Text": "テキスト",
+    "Text color": "文字色",
+    "Text to display": "表示するテキスト",
+    "The URL you entered seems to be an email address. Do you want to add the required mailto: prefix?": "入力された URL は、メールアドレスのようです。mailto: をつけますか？",
+    "The URL you entered seems to be an external link. Do you want to add the required http:// prefix?": "入力されたURLは外部リンクのようです。http:// を付けますか？",
+    "Title Case": "Title Case",
+    "To open the popup, press Shift+Enter": "To open the popup, press Shift+Enter",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:286]:
```suggestion
    "To open the popup, press Shift+Enter": "ポップアップを開くには、Shift+Enterキーを押します",
```

File: locales/ja/wysiwyg.json [hunk 19]
```diff
@@ -0,0 +1,326 @@
...
+    "Paste as text": "Paste as text",
+    "Paste or type a link": "リンクを貼り付けまたは入力",
+    "Paste row after": "下に 1 行貼り付け",
+    "Paste row before": "上に 1 行貼り付け",
+    "Paste your embed code below:": "embed コードを下に貼り付けてください:",
+    "People": "人々",
+    "Permanent Pen Properties": "Permanent Pen Properties",
+    "Permanent pen properties...": "Permanent pen properties...",
+    "Poster": "Poster",
+    "Powered by {0}": "Powered by {0}",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:223]:
```suggestion
    "Powered by {0}": "搭載 {0}",
```

File: locales/ja/wysiwyg.json [hunk 20]
```diff
@@ -0,0 +1,326 @@
...
+    "Paragraph": "段落",
+    "Paste": "貼り付け",
+    "Paste as text": "Paste as text",
+    "Paste or type a link": "リンクを貼り付けまたは入力",
+    "Paste row after": "下に 1 行貼り付け",
+    "Paste row before": "上に 1 行貼り付け",
+    "Paste your embed code below:": "embed コードを下に貼り付けてください:",
+    "People": "人々",
+    "Permanent Pen Properties": "Permanent Pen Properties",
+    "Permanent pen properties...": "Permanent pen properties...",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:221]:
```suggestion
    "Permanent pen properties...": "油性ペンのプロパティ...",
```

File: locales/ja/wysiwyg.json [hunk 21]
```diff
@@ -0,0 +1,326 @@
...
+    "Page break": "改ページ",
+    "Paragraph": "段落",
+    "Paste": "貼り付け",
+    "Paste as text": "Paste as text",
+    "Paste or type a link": "リンクを貼り付けまたは入力",
+    "Paste row after": "下に 1 行貼り付け",
+    "Paste row before": "上に 1 行貼り付け",
+    "Paste your embed code below:": "embed コードを下に貼り付けてください:",
+    "People": "人々",
+    "Permanent Pen Properties": "Permanent Pen Properties",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:220]:
```suggestion
    "Permanent Pen Properties": "油性ペンのプロパティ",
```

File: locales/ja/wysiwyg.json [hunk 22]
```diff
@@ -0,0 +1,326 @@
...
+    "OR": "OR",
+    "Objects": "オブジェクト",
+    "Ok": "Ok",
+    "Open help dialog": "ヘルプダイアログを開く",
+    "Open link in...": "Open link in...",
+    "Orange": "オレンジ",
+    "Page break": "改ページ",
+    "Paragraph": "段落",
+    "Paste": "貼り付け",
+    "Paste as text": "Paste as text",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:214]:
```suggestion
    "Paste as text": "テキストとして貼り付ける",
```

File: locales/ja/wysiwyg.json [hunk 23]
```diff
@@ -0,0 +1,326 @@
...
+    "Middle": "Middle",
+    "Midnight Blue": "Midnight Blue",
+    "More...": "More...",
+    "Name": "名前",
+    "Navy Blue": "ネイビー",
+    "New window": "新しいウインドウ",
+    "Next": "次",
+    "No": "いいえ",
+    "No color": "色なし",
+    "Nonbreaking space": "Nonbreaking space",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:202]:
```suggestion
    "Nonbreaking space": "ノーブレークスペース",
```

File: locales/ja/wysiwyg.json [hunk 24]
```diff
@@ -0,0 +1,326 @@
...
+    "Mathematical": "Mathematical",
+    "Media": "Media",
+    "Media poster (Image URL)": "Media poster (画像 URL)",
+    "Media...": "Media...",
+    "Medium Blue": "Medium Blue",
+    "Medium Gray": "薄めのグレー",
+    "Medium Purple": "Medium Purple",
+    "Merge cells": "セルで統合",
+    "Middle": "Middle",
+    "Midnight Blue": "Midnight Blue",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:194]:
```suggestion
    "Midnight Blue": "ミッドナイトブルー",
```

File: locales/ja/wysiwyg.json [hunk 25]
```diff
@@ -0,0 +1,326 @@
...
+    "Match case": "Match case",
+    "Mathematical": "Mathematical",
+    "Media": "Media",
+    "Media poster (Image URL)": "Media poster (画像 URL)",
+    "Media...": "Media...",
+    "Medium Blue": "Medium Blue",
+    "Medium Gray": "薄めのグレー",
+    "Medium Purple": "Medium Purple",
+    "Merge cells": "セルで統合",
+    "Middle": "Middle",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:193]:
```suggestion
    "Middle": "真ん中",
```

File: locales/ja/wysiwyg.json [hunk 26]
```diff
@@ -0,0 +1,326 @@
...
+    "Link...": "リンク...",
+    "Loading emoticons...": "絵文字を読込中...",
+    "Lower Alpha": "英小文字",
+    "Lower Greek": "ギリシャ小文字",
+    "Lower Roman": "ローマ数字小文字",
+    "Match case": "Match case",
+    "Mathematical": "Mathematical",
+    "Media": "Media",
+    "Media poster (Image URL)": "Media poster (画像 URL)",
+    "Media...": "Media...",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:188]:
```suggestion
    "Media...": "メディア...",
```

File: locales/ja/wysiwyg.json [hunk 27]
```diff
@@ -0,0 +1,326 @@
...
+    "Link": "リンク",
+    "Link list": "リンク一覧",
+    "Link...": "リンク...",
+    "Loading emoticons...": "絵文字を読込中...",
+    "Lower Alpha": "英小文字",
+    "Lower Greek": "ギリシャ小文字",
+    "Lower Roman": "ローマ数字小文字",
+    "Match case": "Match case",
+    "Mathematical": "Mathematical",
+    "Media": "Media",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:186]:
```suggestion
    "Media": "メディア",
```

File: locales/ja/wysiwyg.json [hunk 28]
```diff
@@ -0,0 +1,326 @@
...
+    "Light Yellow": "薄い黄色",
+    "Link": "リンク",
+    "Link list": "リンク一覧",
+    "Link...": "リンク...",
+    "Loading emoticons...": "絵文字を読込中...",
+    "Lower Alpha": "英小文字",
+    "Lower Greek": "ギリシャ小文字",
+    "Lower Roman": "ローマ数字小文字",
+    "Match case": "Match case",
+    "Mathematical": "Mathematical",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:185]:
```suggestion
    "Mathematical": "数学",
```

File: locales/ja/wysiwyg.json [hunk 29]
```diff
@@ -0,0 +1,326 @@
...
+    "Alternative source URL": "代替ソース URL",
+    "Anchor": "アンカー",
+    "Anchor...": "アンカー...",
+    "Anchors": "アンカー",
+    "Animals and Nature": "動物・自然",
+    "Arrows": "矢印",
+    "B": "B",
+    "Background color": "背景色",
+    "Black": "黒",
+    "Block": "Block",
```
Comment (rbenitezpagan) [inline locales/ja/wysiwyg.json:22]:
```suggestion
    "Block": "ブロック",
```

**Comment by rbenitezpagan** [comment]
@fu-sen I definitely don't know Japanese and the suggestions are based on machine translations. Feel free to accept/deny accordingly.

**Comment by fu-sen** [comment]
@rbenitezpagan Thank you for the great app and support.

I think this can be reflected once.
I will continue to be able to make fix pull requests as needed.

**Review by rbenitezpagan — APPROVED** [review]
Package is complete. Revisions will be submitted.

File: locales/ja/wysiwyg.json [hunk 30]
```diff
@@ -0,0 +1,326 @@
...
+    "Bold": "太字",
+    "Border": "Border",
+    "Border color": "Border 色",
+    "Border style": "Border スタイル",
+    "Border width": "Border 幅",
+    "Bottom": "下",
+    "Browse for an image": "画像の閲覧",
+    "Bullet list": "箇条書きリスト",
+    "Cancel": "キャンセル",
+    "Capitalization": "Capitalization",
```
[the remark this answers is not in the document]
Comment (fu-sen) [reply locales/ja/wysiwyg.json:35]: Hmmm, this doesn't make sense. So I left it in English.

File: locales/ja/wysiwyg.json (hunk 1 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:63]: This may have the wrong meaning. 通貨＝＄ € ￥

File: locales/ja/wysiwyg.json (hunk 6 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:305]: This is an incorrect translation.

File: locales/ja/wysiwyg.json (hunk 7 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:315]: I don't understand the meaning of this. "位置"?

File: locales/ja/wysiwyg.json (hunk 9 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:318]: or "書式"

File: locales/ja/wysiwyg.json (hunk 10 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:321]: This corresponds to English characters, so I thought it was better not to use Japanese.

File: locales/ja/wysiwyg.json (hunk 11 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:322]: This is an incorrect translation

File: locales/ja/wysiwyg.json (hunk 14 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:299]: Most likely a funny translation

File: locales/ja/wysiwyg.json (hunk 15 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:300]: Funny translation

File: locales/ja/wysiwyg.json (hunk 16 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:291]: Same as "lowercase". I dared to maintain English.

File: locales/ja/wysiwyg.json (hunk 19 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:223]: It is recommended to maintain English.

File: locales/ja/wysiwyg.json (hunk 20 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:221]: This is a funny translation

File: locales/ja/wysiwyg.json (hunk 21 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:220]: This is a funny translation

File: locales/ja/wysiwyg.json (hunk 23 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:202]: I don't understand the meaning

File: locales/ja/wysiwyg.json (hunk 24 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:194]: I want a different translation.

File: locales/ja/wysiwyg.json (hunk 25 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:193]: What is this used for? The meaning can be strange.

**Comment by fu-sen** [comment]
Please note that some items dare to maintain English.

File: locales/ja/wysiwyg.json (hunk 10 above)
Comment (rbenitezpagan) [reply locales/ja/wysiwyg.json:321]: Makes sense. Same with Uppercase.

File: locales/ja/wysiwyg.json (hunk 7 above)
Comment (rbenitezpagan) [reply locales/ja/wysiwyg.json:315]: "位置" translates to "position" so it could work.

File: locales/ja/wysiwyg.json (hunk 25 above)
Comment (rbenitezpagan) [reply locales/ja/wysiwyg.json:193]: I believe is in respect to the position of the element. "Middle" and "Center" are kind of synonyms.

File: locales/ja/wysiwyg.json (hunk 12 above)
Comment (fu-sen) [reply locales/ja/wysiwyg.json:302]: It's correct, but I dared to leave it in English.

**Comment by rbenitezpagan** [comment]
@fu-sen I did notice that you have the old wrong keys with typos. Can you make sure to update the typos in your submission?

https://github.com/GetPublii/Publii-ui-locales/blob/main/diff-files/v.0.40.0.diff

**Review by rbenitezpagan — CHANGES_REQUESTED** [review]
Fixed typos in keys updated from `v0.39.0` to `v0.40.0`.

https://github.com/GetPublii/Publii-ui-locales/blob/main/diff-files/v.0.40.0.diff

**Comment by fu-sen** [comment]
that? I intended to reflect this. Like "whatsNew" and so on.

**Comment by rbenitezpagan** [comment]
@fu-sen 

Example
```
#Typo
"file": {
  "renameBackupConfrimMsg"
}
#Fix
"file": {
  "renameBackupConfirmMsg"
}
```

## Typos

'file.renameBackupConfrimMsg', 'post.configureThemeBeforeGenaratingPreview', 'file.deletBackupsSuccessMsg', 'menu.menuIemsRemoveMessage', 'file.deletBackupsErrorMsg', 'file.selectedFileExsistsMsg', 'file.restoreBackupConfrimMsg', 'sync.testConnectionNoPassowrdMsg', 'file.restoreBackupConfrimLabel', 'file.renameBackupConfrimLabel', 'file.deletBackupsConfirmMsg']

## Fixes

['menu.menuItemsRemoveMessage', 'file.deleteBackupsSuccessMsg', 'file.deleteBackupsConfirmMsg', 'file.restoreBackupConfirmLabel', 'file.renameBackupConfirmMsg', 'post.configureThemeBeforeGeneratingPreview', 'file.renameBackupConfirmLabel', 'file.selectedFileExistsMsg', 'file.deleteBackupsErrorMsg', 'file.restoreBackupConfirmMsg', 'sync.testConnectionNoPasswordMsg', 'file.renameBackupConfrimMsg', 'post.configureThemeBeforeGenaratingPreview', 'file.deletBackupsSuccessMsg', 'menu.menuIemsRemoveMessage', 'file.deletBackupsErrorMsg', 'file.selectedFileExsistsMsg', 'file.restoreBackupConfrimMsg', 'sync.testConnectionNoPassowrdMsg', 'file.restoreBackupConfrimLabel', 'file.renameBackupConfrimLabel', 'file.deletBackupsConfirmMsg']

**Comment by fu-sen** [comment]
Some of the items were found. This is the diff:
https://github.com/fu-sen/Publii-ui-locales-Japanese/commit/77d58c772fc9107db160d7cf16adb7ac616de41e

**Comment by rbenitezpagan** [comment]
@fu-sen Awesome.  

- [x] Update keys' typos
- [ ] Fix the indentation in `config.json` (Should be using spaces not the tab, that was also fixed, not your fault.)
- [ ] Create new .zip file
- [ ] Submit new PR

I suggest closing this PR and creating a new one (so it doesn't have all my comments).

**Pull request merged**